Chell0/omnifood
# Build the landing page

## What does this PR do?
Have the landing page of the website.
## Description of Task to be completed?
Have the landing page appear when the website first loads.
## How should this be manually tested?
- Clone this repo.
- cd into the folder.
- Use your preferred browser to launch the HTML file.
- Voila!  The landing page appears.
## Screenshots
![landing-page](https://user-images.githubusercontent.com/15849070/52112494-bb4d0100-2617-11e9-9707-9d01aac17e0f.jpg)
